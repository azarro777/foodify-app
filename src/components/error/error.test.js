import React from 'react';
import Error from './error';
import { shallow } from 'enzyme';

describe('Testing <Error/> component', () => {
  const error = shallow(<Error />);
  it('Error have rendered correctly', () => {
    expect(error).toMatchSnapshot();
  });
});
