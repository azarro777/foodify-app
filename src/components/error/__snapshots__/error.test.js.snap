// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Error/> component Error have rendered correctly 1`] = `ShallowWrapper {}`;
